nodejs/week2/nodejs_week2_homework/app.js

```diff
@@ -0,0 +1,154 @@
+import express from "express";
+const app = express();
+const port = process.env.PORT || 3000;
+
+import fs from "fs";
+const loadDocuments = () =>
+  JSON.parse(fs.readFileSync("documents.json", "utf-8"));
+
```

Comment:
Please try to read async way , it's okay when reading a small file , but for large this could block entire event loop.
ref -> https://nodejs.org/en/learn/manipulating-files/reading-files-with-nodejs